### PR TITLE
Docs: TAG policy revision 

### LINKED ISSUE #41931: Docs: TAG policy revision
## Applies To

Docs: https://goteleport.com/docs/access-controls/access-graph/
Site: https://goteleport.com/platform/policy/

## Details

Modify the existing Access Graph docs to frame TAG as the Policy product 

## How will we know this is resolved?

When this is updated accordingly 

## Related Issues

This came up as part of the docs site reorganization: https://github.com/gravitational/teleport.e/issues/4117

### PR DESCRIPTION
**Pull request by mmcallister**
Issue https://github.com/gravitational/teleport/issues/41931
Reframe TAG as the Policy product

### CODE REVIEW

**Comment by mmcallister** [comment]
Moving to draft status. Chatted with Paul, making a couple of other changes.

**Comment by ptgott** [comment]
@mmcallister Let me know when this is ready for me to take a look!

File: docs/pages/access-controls/access-graph.mdx [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-description: A reference for Teleport Access Graph.
+title: Teleport Policy
+description: A reference for Access Graph with Teleport Policy.
 ---
 
-Teleport Access Graph visualizes and helps you understand access to your
-infrastructure. It provides a visual representation of the relationships between
-users, roles, and resources in your organization. It can help you answer
-questions like:
+Teleport Policy will streamline and centralize access management across your entire infrastructure.
```
Comment (ptgott) [inline docs/pages/access-controls/access-graph.mdx:6]:
```suggestion
Teleport Policy streamlines and centralizes access management across your entire infrastructure.
```

So this sounds less like a promise and more like a feature description

File: docs/pages/access-controls/access-graph/aws-sync.mdx [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+title: Discover AWS Access Patterns with Teleport Policy
+description: Describes how to import and visualize AWS accounts access patterns using Teleport Policy and Access Graph.
 ---
 
-Teleport Access Graph offers insights into access patterns within
-your AWS account. By scanning IAM permissions, users, groups, resources, and 
-identities, it provides a visual representation and aids in enhancing the 
-permission model within your AWS environment. This functionality enables you
-to address queries such as:
+Teleport Policy will streamline and centralize access management across your entire infrastructure. You can view access relationships in seconds,
```
Comment (ptgott) [inline docs/pages/access-controls/access-graph/aws-sync.mdx:6]:
```suggestion
Teleport Policy streamlines and centralizes access management across your entire infrastructure. You can view access relationships in seconds, 
```

File: docs/pages/access-controls/access-graph/aws-sync.mdx [hunk 3]
```diff
@@ -76,8 +75,8 @@ from Teleport Auth Service and Discovery Service.
 
 <Notice type="warning">
 
-If you have a Teleport Cloud cluster, you can disregard
-this step, as Teleport Cloud already operates a properly configured 
+If you have a Teleport Enterprise cluster, you can disregard
```
Comment (ptgott) [inline docs/pages/access-controls/access-graph/aws-sync.mdx:78]:
```suggestion
If you have a managed Teleport Enterprise cluster, you can disregard
```

File: docs/pages/access-controls/access-graph/aws-sync.mdx [hunk 4]
```diff
@@ -76,8 +75,8 @@ from Teleport Auth Service and Discovery Service.
...
+this step, as Teleport Enterprise already operates a properly configured
```
Comment (ptgott) [inline docs/pages/access-controls/access-graph/aws-sync.mdx:79]:
```suggestion
this step, as managed Teleport Enterprise already operates a properly configured 
```
Self-hosted Teleport Enterprise would still require you to deploy Access Graph

**Review by ptgott — APPROVED** [review]

**Review by klizhentas — APPROVED** [review]

**Pull request merged**